### PR TITLE
update using.rst

### PR DESCRIPTION
**Pull request by ashish-timilsina**

### CODE REVIEW

File: docs/source/admin/traffic_ops/using.rst [hunk 1]
```diff
@@ -345,9 +347,10 @@ The fields in the Delivery Service view are:
...
+| IPv6 Routing Enabled                         | Default is "True", entering "False" allows you to turn off CDN response to IPv6 requests                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (mitchell852) [inline docs/source/admin/traffic_ops/using.rst]: It looks like the default for this is actually false

image

image

File: docs/source/admin/traffic_ops/using.rst [hunk 2]
```diff
@@ -409,23 +412,26 @@ The fields in the Delivery Service view are:
...
+| Long Description 1                           | Free text field not currently used in configuration. For example, you can use this field to describe your customer type. To be consumed from the APIs by downstream tools such as the Traffic Portal                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (mitchell852) [inline docs/source/admin/traffic_ops/using.rst]: can you get rid of the "such as the Traffic Portal"? Traffic Portal doesn't really use this field, instead it allows you to set this field. That is it.

File: docs/source/admin/traffic_ops/using.rst [hunk 3]
```diff
@@ -409,23 +412,26 @@ The fields in the Delivery Service view are:
...
+| Long Description 2                           | Free text field not currently used in configuration. To be consumed from the APIs by downstream tools such as the Traffic Portal                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (mitchell852) [inline docs/source/admin/traffic_ops/using.rst]: can you get rid of the "such as the Traffic Portal"? Traffic Portal doesn't really use this field, instead it allows you to set this field. That is it.

File: docs/source/admin/traffic_ops/using.rst [hunk 4]
```diff
@@ -409,23 +412,26 @@ The fields in the Delivery Service view are:
...
+| Info URL                                     | Free text field allowing you to enter a URL which provides information about the service. To be consumed from the APIs by downstream tools such as the Traffic Portal                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (mitchell852) [inline docs/source/admin/traffic_ops/using.rst]: can you get rid of the "such as the Traffic Portal"? Traffic Portal doesn't really use this field, instead it allows you to set this field. That is it.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1482/
Test PASSed.

**Pull request merged**